Documentation/policy.rst

```diff
@@ -0,0 +1,408 @@
+.. _arch_guide:
+
+####################
+Policy Specification
+####################
+
+This chapter specifies the policy language used to configure network policies
+in Cilium. Security policies can be specified and imported via the following
+mechanisms:
+
+* The Kubernetes *NetworkPolicy* specification which offers to configure a
+  subset of the full Cilium security. For fun see `Kubernetes Network Policies
```

Comment:
'a subset of the full Cilium security' is awkwardly phrased. What about "which configures a subset of what Cilium can secure"?